hack/images/dns/kube2sky/kube2sky.go

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
Having the code vendored in one place might make it easier to keep up-to-date.  Our goal is to un-vendor as much as possible, so If it's not too much pain, could we avoid bringing this copy into the source tree?